awx_collection/tools/roles/template_galaxy/templates/README.md.j2

```diff
@@ -52,12 +69,14 @@ Notable releases of the `{{ collection_namespace }}.{{ collection_package }}` co
  - 11.0.0 has no non-deprecated modules that depend on the deprecated `tower-cli` [PyPI](https://pypi.org/project/ansible-tower-cli/).
  - 19.2.1 large renaming purged "tower" names (like options and module names), adding redirects for old names
  - 21.11.0 "tower" modules deprecated and symlinks removed.
+ - 25.0.0 "token" module has been removed as oauth is no longer supported, use basic auth instead
```

Comment:
Same question here about the application module.

Reply:
Updated, nice catch!